iphone/Classes/TiUIView.m

```diff
@@ -1046,11 +1046,11 @@ -(UITapGestureRecognizer*)singleTapRecognizer
 		singleTapRecognizer = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(recognizedTap:)];
 		[self configureGestureRecognizer:singleTapRecognizer];
 		[self addGestureRecognizer:singleTapRecognizer];
-
 		if (doubleTapRecognizer != nil) {
 			[singleTapRecognizer requireGestureRecognizerToFail:doubleTapRecognizer];
 		}
 	}
+    singleTapRecognizer.delegate = self;
```

Comment:
Indentation here and all other places where you assign this delegate incorrect, please adjust.